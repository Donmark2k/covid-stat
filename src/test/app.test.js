// import React from 'react';
// import renderer from 'react-test-renderer';
// import { BrowserRouter } from 'react-router-dom';
// import { Provider } from 'react-redux';
// import { store } from '../redux/store';
// import App from '../App';
// import NavBar from '../components/navbar';

// it('App renders correctly', () => {
//   const tree = renderer
//     .create(
//       <React.StrictMode>
//         <Provider store={store}>
//           <BrowserRouter>
//             <App />
//           </BrowserRouter>
//         </Provider>
//       </React.StrictMode>,
//     )
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// });

// it('NavBar page renders correctly', () => {
//   const tree = renderer
//     .create(
//       <React.StrictMode>
//         <Provider store={store}>
//           <BrowserRouter>
//             <NavBar />
//           </BrowserRouter>
//         </Provider>
//       </React.StrictMode>,
//     )
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// });
