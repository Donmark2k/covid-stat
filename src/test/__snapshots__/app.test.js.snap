// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="App"
>
  <div>
    <nav
      className="px-5 navbar navbar-expand-lg navbar-light bg-white"
    >
      <a
        className="fs-2 fw-medium navbar-brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="planet log"
          height={55}
          src="planet.png"
          width={55}
        />
          
        Space Traveler's Hub
      </a>
      <button
        aria-controls="basic-navbar-nav "
        aria-label="Toggle navigation"
        className="navbar-toggler collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        aria-expanded={null}
        className="justify-content-end navbar-collapse collapse"
        id="basic-navbar-nav"
      >
        <div
          className="fs-5 navList navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            aria-current="page"
            as={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "render": [Function],
              }
            }
            className="nav-link active"
            href="/"
            onClick={[Function]}
          >
            Rockets
          </a>
          <a
            as={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "render": [Function],
              }
            }
            className="nav-link"
            href="/missions"
            onClick={[Function]}
          >
            Missions
          </a>
          <div
            className="d-none d-md-none d-lg-block mt-3 mb-3 vr"
          />
          <a
            as={
              Object {
                "$$typeof": Symbol(react.forward_ref),
                "render": [Function],
              }
            }
            className="nav-link"
            href="/profile"
            onClick={[Function]}
          >
            My profile
          </a>
        </div>
      </div>
    </nav>
    <hr />
  </div>
</div>
`;

exports[`NavBar page renders correctly 1`] = `
<div>
  <nav
    className="px-5 navbar navbar-expand-lg navbar-light bg-white"
  >
    <a
      className="fs-2 fw-medium navbar-brand"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="planet log"
        height={55}
        src="planet.png"
        width={55}
      />
        
      Space Traveler's Hub
    </a>
    <button
      aria-controls="basic-navbar-nav "
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="justify-content-end navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="fs-5 navList navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          aria-current="page"
          as={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            }
          }
          className="nav-link active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
        <a
          as={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            }
          }
          className="nav-link"
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
        <div
          className="d-none d-md-none d-lg-block mt-3 mb-3 vr"
        />
        <a
          as={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            }
          }
          className="nav-link"
          href="/profile"
          onClick={[Function]}
        >
          My profile
        </a>
      </div>
    </div>
  </nav>
  <hr />
</div>
`;
